draft-irtf-cfrg-frost.md

```diff
@@ -865,6 +868,18 @@ signature under the group public key. However, since the aggregate signature is
 and only if all signature shares are valid, this order of operations is more expensive
 if the signature is valid.
 
+## Identifiable Abort {#abort}
+
+FROST does not provide robustness. As such, a misbehaving participant can cause a denial-of-service
+on the signing protocol if it is allowed to contribute to the signing protocol.
+Preventing this type of attack requires the Coordinator to identify the misbehaving
+participant such that applications can take corrective action to prevent denial-of-service.
```

Comment:
```suggestion
participant such that applications can take corrective action.
```

Reply:
I don't agree with this suggestion since this is the purpose of aborting. That is, DoS resistance is the security property we get from identifiable aborts.